gcp_variant_transforms/libs/bigquery_vcf_schema.py

```diff
@@ -258,17 +264,26 @@ def get_rows_from_variant(variant, split_alternate_allele_info_fields=True):
 
 
 def _get_call_record(call):
-  """A helper method for ``get_rows_from_variant`` to get a call as JSON."""
+  """A helper method for ``get_rows_from_variant`` to get a call as JSON.
+
+  Args:
+    call: PyVCF Variant call to convert.
```

Comment:
nit: this is actually  ` ``VariantCall`` ` inside vcfio.
Please use:
call (` ``VariantCall`` `): Variant call to convert.